crates/turborepo-repository/src/package_json.rs

```diff
@@ -159,7 +159,7 @@ impl From<RawPnpmConfig> for PnpmConfig {
 
 impl PackageJson {
     pub fn load(path: &AbsoluteSystemPath) -> Result<PackageJson, Error> {
-        tracing::debug!("loading package.json from {}", path);
+        tracing::trace!("loading package.json from {}", path);
```

Comment:
This is call N times in a monorepo with N packages, making `-vv` logging very noisy. I think it's better to put this in `-vvv` (trace) logging